commands/start.go

```diff
@@ -0,0 +1,68 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/ipfs/iptb/testbed"
+	"github.com/ipfs/iptb/testbed/interfaces"
+
+	cli "github.com/urfave/cli"
+)
+
+var StartCmd = cli.Command{
+	Category:  "CORE",
+	Name:      "start",
+	Usage:     "start specified nodes (or all)",
+	ArgsUsage: "[nodes] -- [arguments...]",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "wait",
+			Usage: "wait for nodes to start before returning",
+		},
+		cli.BoolFlag{
+			Name:   "terminator",
+			Hidden: true,
+		},
+	},
+	Before: func(c *cli.Context) error {
+		if present := isTerminatorPresent(c); present {
```

Comment:
let's leave a comment describing what `terminator` means

Reply:
I added a comment on `isTerminatorPresent` as this is used on any command which takes a node range.